LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Shields.io Team
```

Comment:
Since Shields.io isn't a legal entity I think it would be better to put some real names here. I'd suggest:

> Copyright (c) 2020 Caleb Cartwright and other contributors

Reply:
no objections to doing so, though was hoping for more of a collective thing than an individual. What do we exist as under OpenCollective? (that sounds like such a weird question :laughing:)

Reply:
(have updated by the way, i just keep forgetting how OC works)

Reply:
We're a project sponsored by their organization. I don't think we want to assign the copyright to Open Collective.

In the long run I think the thing to do is create an unincorporated association that can function as a legal governing organization for Shields. It's pretty easy to set up, and as long as we don't handle any money, there is very little administrative burden. Then we could continue to use Open Collective as a fiscal sponsor to handle our $. Although if you wanted the Shields Association to own your contributions instead of you, you'd still have to assign them. But at least you'd have that option.

Reply:
> We're a project sponsored by their organization. I don't think we want to assign the copyright to Open Collective.

Makes sense, thanks for the refresher (err the 8th refresher lol)